assets/src/blocks/Splittwocolumns/SplittwocolumnsEditor.js

```diff
@@ -30,20 +35,20 @@ export const SplittwocolumnsEditor = ({ attributes, setAttributes, isSelected })
 
   return (
     isSelected
-      ? renderEdit({attributes}, setAttributes) 
+      ? renderEdit({attributes}, setAttributes)
       : renderView({attributes})
   );
 }
 
-const renderView = ({attributes}) => <Frontend {...attributes} />
+const renderView = ({attributes}) => <SplittwocolumnsFrontend {...attributes} />
 const renderEdit = ({attributes}, setAttributes) => {
   const charLimit = { title: 40, description: 400 };
   const params = {attributes, charLimit, setAttributes};
 
   return (
     <Fragment>
-      <InPlaceEdit {...params} />
-      <SidebarSettings  {...params} />
+      <SplittwocolumnsInPlaceEdit {...params} />
```

Comment:
I removed the alias here as they make the code significantly harder to read and navigate. Another option would be to just call the component `InPlaceEdit` in the first place.